ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -3,6 +3,10 @@
 
 @implementation CodePushUpdateUtils
 
+NSString * const ManifestFolderPrefix = @"CodePush";
+NSString * const DefaultJsBundleName = @"main.jsbundle";
```

Comment:
My thought was, even if they rename it locally, the one deployed to our server will be called `main.jsbundle` if they use the `release-react` command, so in order to generate a matching hash we should rename it for them in the manifest.